src/llamacrawl/cli/utils.py

```
@@ -0,0 +1,106 @@
+Functions:
+    validate_url: Validate HTTP(S) URL format
+    read_schema_file: Read JSON schema from file path
+    run_cli_coroutine: Execute async coroutine from sync CLI code
+"""
+
+from __future__ import annotations
+
+import asyncio
+from collections.abc import Coroutine
+from pathlib import Path
+from typing import Awaitable, TypeVar, cast
```

Ruff (0.14.0)

Import from `collections.abc` instead: `Awaitable`

Import from `collections.abc`

(UP035)